### PR TITLE
#2579 RPK updates for changes to RP Connect commands

### PR DESCRIPTION
**Pull request by asimms41**
## Description

Resolves https://github.com/redpanda-data/documentation-private/issues/2579
Review deadline: 12 July 

## Page previews

[rpk connect](https://deploy-preview-598--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-connect/rpk-connect/)
[rpk connect echo](https://deploy-preview-598--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-connect/rpk-connect-echo/)
[rpk connect lint](https://deploy-preview-598--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-connect/rpk-connect-lint/)
[rpk connect streams](https://deploy-preview-598--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-connect/rpk-connect-echo/)

## Checks

- [ ] New feature
- [ ] Content gap
- [x] Support Follow-up
- [x] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

File: modules/reference/pages/rpk/rpk-connect/rpk-connect-streams.adoc [hunk 1]
```diff
@@ -20,7 +20,7 @@ rpk connect streams
 
 [,bash]
 ----
-rpk connect -c ./root_config.yaml streams`
+rpk connect -c ./root_config.yaml streams
```
Comment (Jeffail) [inline modules/reference/pages/rpk/rpk-connect/rpk-connect-streams.adoc]:
```suggestion
rpk connect streams -o ./root_config.yaml 
```

File: modules/reference/pages/rpk/rpk-connect/rpk-connect-lint.adoc [hunk 2]
```diff
@@ -14,15 +14,15 @@ rpk connect lint [command options] [arguments...]
 == Examples
 
 ```bash
-rpk connect -c target.yaml lint
+rpk connect lint -c target.yaml
```
Comment (Jeffail) [inline modules/reference/pages/rpk/rpk-connect/rpk-connect-lint.adoc]:
```suggestion
rpk connect lint target.yaml
```

File: modules/reference/pages/rpk/rpk-connect/rpk-connect-lint.adoc [hunk 3]
```diff
@@ -35,6 +35,10 @@ rpk connect lint ./configs/...
 |===
 |*Value* |*Type* |*Description*
 
+|--config, -c |- | Path to a configuration file.
```
Comment (Jeffail) [inline modules/reference/pages/rpk/rpk-connect/rpk-connect-lint.adoc]: This flag is removed.

File: modules/reference/pages/rpk/rpk-connect/rpk-connect-streams.adoc (hunk 1 above)
Comment (asimms41) [reply modules/reference/pages/rpk/rpk-connect/rpk-connect-streams.adoc]: Updated.

File: modules/reference/pages/rpk/rpk-connect/rpk-connect-lint.adoc (hunk 2 above)
Comment (asimms41) [reply modules/reference/pages/rpk/rpk-connect/rpk-connect-lint.adoc]: Updated.

File: modules/reference/pages/rpk/rpk-connect/rpk-connect-lint.adoc (hunk 3 above)
Comment (asimms41) [reply modules/reference/pages/rpk/rpk-connect/rpk-connect-lint.adoc]: Updated.

**Review by JakeSCahill — COMMENTED** [review]
These docs are autogenerated. Have these changes been made in the `rpk` source code?

**Comment by asimms41** [comment]
> These docs are autogenerated. Have these changes been made in the `rpk` source code?

**Review by Jeffail — APPROVED** [review]

**Review by JakeSCahill — APPROVED** [review]

**Pull request merged**